#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-recording-save.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: System recordings should be done in the custom sounds dir
## DP: and not in the /tmp/. This simplifies the code and works better
## DP: To be commited upstream "soon" and removed in version 2.2.3 or 2.3.0

@DPATCH@
diff -urNad upstream~/amp_conf/astetc/extensions.conf upstream/amp_conf/astetc/extensions.conf
--- upstream~/amp_conf/astetc/extensions.conf	2007-06-08 05:50:53.000000000 +0300
+++ upstream/amp_conf/astetc/extensions.conf	2007-06-27 14:41:34.000000000 +0300
@@ -673,11 +673,11 @@
 [macro-systemrecording]
 exten => s,1,Goto(${ARG1},1)
 
-exten => dorecord,1,Record(/tmp/${CALLERID(number)}-ivrrecording:wav)
+exten => dorecord,1,Record(custom/${CALLERID(number)}-ivrrecording:wav)
 exten => dorecord,n,Wait(1)
 exten => dorecord,n,Goto(confmenu,1)
 
-exten => docheck,1,Playback(/tmp/${CALLERID(number)}-ivrrecording)
+exten => docheck,1,Playback(custom/${CALLERID(number)}-ivrrecording)
 exten => docheck,n,Wait(1)
 exten => docheck,n,Goto(confmenu,1)
 
